Source/Orts.Formats.Msts/WagonFile.cs

```diff
@@ -27,7 +27,32 @@ namespace Orts.Formats.Msts
     /// </summary>
     public class WagonFile
     {
+        public class CarSize
+        {
+            public float CarWidth;
+            public float CarHeight;
+            public float CarLength;
+
+            public CarSize(STFReader stf)
+            {
+                stf.MustMatch("(");
+                CarWidth = stf.ReadFloat(STFReader.UNITS.Distance, null);
+                CarHeight = stf.ReadFloat(STFReader.UNITS.Distance, null);
+                CarLength = stf.ReadFloat(STFReader.UNITS.Distance, null);
+                stf.SkipRestOfBlock(); // or should this be stf.MustMatch(")")
```

Comment:
Which would be better here, SkipRestOfBlock or MustMatch?

Reply:
If we have no reason to expect more data in the block, I'd lean slightly towards `MustMatch()` simply so that it'll print a warning message if it **isn't** the end of the block.

Reply:
Thanks for the feedback. Changed to use MustMatch.